test/onnx/exporter/test_small_models_e2e.py

```diff
@@ -29,6 +34,13 @@ def export(self, model, args=(), kwargs=None, **options) -> torch.onnx.ONNXProgr
         assert onnx_program is not None
         return onnx_program
 
+    def has_onnxruntime_opset_23(self) -> bool:
+        try:
```

Comment:
I think we can skip the try import part. We always assume onnxruntime in these tests